.rubocop_todo.yml

```diff
@@ -464,13 +464,6 @@ Style/TernaryParentheses:
     - 'lib/fastlane/plugin/wpmreleasetoolkit/actions/common/extract_release_notes_for_version_action.rb'
     - 'lib/fastlane/plugin/wpmreleasetoolkit/helper/configure_helper.rb'
 
-# Offense count: 20
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyleForMultiline.
-# SupportedStylesForMultiline: comma, consistent_comma, no_comma
-Style/TrailingCommaInArrayLiteral:
```

Comment:
I'm surprised by this one because typically we'd leave it in case we needed to add another entry later.

I presume this was just the less-destructive approach?

Reply:
It was. But good point, we could switch the rule the other way and require them all the time, I'm fine with it too, will see how much changes it would make to turn it on.

Reply:
rule flipped in f6499f759a22dca36b72b136bca963c66b9f25cf, agree it's nicer that way